charts/sps-sample/values.yaml

```diff
@@ -8,7 +8,7 @@ refarch-gateway:
     - name: SPRING_CLOUD_GATEWAY_ROUTES_0_ID
       value: backend
     - name: SPRING_CLOUD_GATEWAY_ROUTES_0_URI
-      value: "http://a-backend:8080/"
+      value: "http://{{ include \"frontend.fullname\" $ }}:8080/"
```

Comment:
Das soll doch die Route zum Backend sein. Ich nehme mal an, das hast du bei der falschen Gateway-Route eingehängt und sollte eigentlich unter SPRING_CLOUD_GATEWAY_ROUTES_1_URI stehen?

Was da rauskommt ist beim Aufruf von `helm template a . -f values.yaml`:

`value: http://a-haus:8080/`

Erwarten würde ich 

`value: http://a-frontend:8080/`

Reply:
sorry habe mich verschriebn
```suggestion
      value: "http://{{ include \"frontend.fullname\" $.Subcharts.haus }}:8080/"
```

Reply:
Ich hab den neuen Vorschlag lokal getestet, da bekomme ich aber einen Fehler:

`Error: template: sps-sample/charts/haus/templates/deployment.yaml:53:16: executing "sps-sample/charts/haus/templates/deployment.yaml" at <tpl (toYaml .Values.env) .>: error calling tpl: error during tpl function execution for "- name: SPRING_PROFILES_ACTIVE\n  value: no-security\n- name: SSO_ISSUER_URL\n  value: https://sso.muenchen.de/auth/realms/muenchen.de\n- name: SPRING_CLOUD_GATEWAY_ROUTES_0_ID\n  value: backend\n- name: SPRING_CLOUD_GATEWAY_ROUTES_0_URI\n  value: http://{{ include \"frontend.fullname\" $.Subcharts.haus }}:8080/\n- name: SPRING_CLOUD_GATEWAY_ROUTES_0_PREDICATES_0\n  value: Path=/api/beispielprojekt-backend-service/**\n- name: SPRING_CLOUD_GATEWAY_ROUTES_0_FILTERS_0\n  value: RewritePath=/api/beispielprojekt-backend-service/(?<urlsegments>.*), /$\\{urlsegments}\n- name: SPRING_CLOUD_GATEWAY_ROUTES_0_FILTERS_1\n  value: RemoveResponseHeader=WWW-Authenticate\n- name: SPRING_CLOUD_GATEWAY_ROUTES_1_ID\n  value: frontend\n- name: SPRING_CLOUD_GATEWAY_ROUTES_1_URI\n  value: http://a-frontend:8080/\n- name: SPRING_CLOUD_GATEWAY_ROUTES_1_PREDICATES_0\n  value: Path=/**\n- name: SPRING_CLOUD_GATEWAY_ROUTES_1_FILTERS_0\n  value: RewritePath=/(?<urlsegments>.*), /$\\{urlsegments}\n- name: SPRING_CLOUD_GATEWAY_ROUTES_1_FILTERS_1\n  value: RemoveResponseHeader=WWW-Authenticate": template: gotpl:8:19: executing "gotpl" at <include "frontend.fullname" $.Subcharts.haus>: error calling include: template: sps-sample/charts/frontend/templates/_helpers.tpl:18:16: executing "frontend.fullname" at <$name>: invalid value; expected string`

Vielleicht hab ich einen Typo. Kannst du das einfach im Branch ändern und bei dir testen? Danke

Reply:
![grafik](https://github.com/user-attachments/assets/bb0ff8b1-85a3-4ffc-894c-99bf4f15441b)


Ich habe es nur in der ApplikationYml hinbekommen, die dann in der ConfigMap im ParentChart liegt

Reply:
Klingt für mich ziemlich unübersichtlich und fehleranfällig.